bin/filter_maf.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import csv
+import sys
+
+input_maf  = sys.argv[1]
+output_maf = sys.argv[2]
+
+with (
+    open(input_maf, newline='') as infile,
+    open(output_maf, "w", newline='') as outfile
+):
+    reader = csv.DictReader(infile, delimiter='\t')
+    writer = csv.DictWriter(outfile, reader.fieldnames, delimiter='\t')
+    writer.writeheader()
+    for row in reader:
+        if row['FILTER'] == 'PASS':
```

Comment:
If you are to filter via the gnomad columns, here is some [code](https://github.com/Sage-Bionetworks/Genie/blob/aaea48c8f63d037a6a9ac03e353944d5dca04751/genie/database_to_staging.py#L312-L323) that we use in GENIE to do so:.

Reply:
Yeah, this is already being done by vcf2maf [here](https://github.com/Sage-Bionetworks-Workflows/vcf2maf/blob/f5dd7284e06b836ece52d1f4e5c594d6e7454c08/vcf2maf.pl#L886-L899).